lib/clamp-device.in

```diff
@@ -210,18 +211,6 @@ if [ $RETVAL != 0 ]; then
   exit $RETVAL
 fi
 
-# Optimization notes:
-#  -disable-simplify-libcalls:  prevents transforming loops into library calls such as memset, memcopy on GPU
-$OPT -mtriple amdgcn-amd-amdhsa -mcpu=$AMDGPU_TARGET \
-  -amdgpu-internalize-symbols -disable-simplify-libcalls $KMOPTOPT -verify \
```

Comment:
The combined `opt` invocation doesn't explicitly specify `amdgpu-internalize-symbols`, does it still run it?

Reply:
LC already runs that anyway. I think we were just doing work that LC does for no clear reason / for historical reasons lost to time. We should really only run the passes that are specific to us, IMHO, possibly without any optimisation, and then let LC's pipeline take care of everything.